include/manager.hpp

```diff
@@ -181,6 +183,18 @@ class Manager
 
     // Shared pointer to worker class
     std::shared_ptr<Worker> m_worker;
+
+    enum class VPDStatus
+    {
+        NotStarted,
+        InProgress,
+        Completed
+    };
+
+    std::unordered_map<VPDStatus, std::string> VPDStatusMap = {
```

Comment:
As of now, I don't think we need this redirection. Setting in place should be sufficient.
Rest looks ok to me.

Reply:
Done